cmd/contour/servecontext.go

```diff
@@ -131,6 +132,14 @@ type serveContext struct {
 
 	// Name of the envoy service to inspect for Ingress status details.
 	EnvoyServiceName string `yaml:"envoy-service-name,omitempty"`
+
+	// DefaultHTTPVersions defines the default set of HTTPS
+	// versions the proxy should accept. HTTP versions are
+	// strings of the form "HTTP/xx". Supported versions are
+	// "HTTP/1.1" and "HTTP/2".
+	//
+	// If this field not specified, all supported versions are accepted.
+	DefaultHTTPVersions []string `yaml:"default-http-versions"`
```

Comment:
"Default" seems like the wrong term here, it's more like "Enabled" since this is overriding the "supported" versions.

Reply:
The intention is to set the default HTTP versions for all the proxy objects. Later we can add per-proxy versions to the `VirtualHost` spec to override these defaults. This also gives us the space to add something like "enabled versions", which would restrict what can become the defaults.